src/frontend/js/hooks/useCourseProductUnion/index.ts

```diff
@@ -23,27 +24,31 @@ const messages = defineMessages({
   },
 });
 
-interface UserOrderEnrollmentsProps {
+interface UseCourseProductUnionProps {
   perPage?: number;
+  organizationId?: string;
 }
 
-export const useCourseProductUnion = ({ perPage = 50 }: UserOrderEnrollmentsProps = {}) => {
+export const useCourseProductUnion = ({
+  perPage = 50,
+  organizationId,
+}: UseCourseProductUnionProps = {}) => {
   const api = useJoanieApi();
   return useUnionResource<
     CourseListItem,
     CourseProductRelation,
-    PaginatedResourceQuery,
-    PaginatedResourceQuery
+    CourseQueryFilters,
```

Comment:
No longer need PaginatedResourceQuery?

Reply:
`per_page` is set in [fetchEntity](https://github.com/openfun/richie/blob/master/src/frontend/js/hooks/useUnionResource/utils/fetchEntity.ts#L45-L49).
I feel like `CourseQueryFilters` should extends `PaginatedResourceQuery` but it seems like typescript handle it as is.
I don't know if this is the right pattern to use, i launche a slack discussion about it.